docs/designers/language-modifiers/language-modifier-raw.md

```diff
@@ -0,0 +1,8 @@
+# raw
```

Comment:
This file needs to be added to `/mkdocs.yml` just above 'regex_replace'.

Reply:
OK, I'll do that quickly.

About the release, I would say that a minor version should be enough; I don't know if anybody if really using auto-escaping and `|escape` at the same time. On the other hand, it's true this is a deep change of behaviour…
I don't know, I let you choose ^^'